### ISSUE TITLE
Project maintainers should still be able to access dashboard when flagged spam

### ISSUE THREAD

**Issue by ericholscher** [readthedocs/readthedocs.org#9990]
Currently when we mark a project as spam, we block the dashboard. This currently blocks the dashboard for the project maintainer as well, so they can't actually go in and fix any issue related to invalid spam classifications. We should at least allow the dashboard access for admin users, but perhaps any logged in user, with a big flag at the top, instead of a full block?

**Comment by agjohnson** [comment]
I think we *do* want dashboard access disabled for projects that cross some spam value threshold though, no? To allow access to a high spam score project, we might be inviting more abuse or at least fiddling with spam projects.

This issue here feels more like we need to adjust the spam rules, and we put too much weight into the invalid repo rule.

The fact that we had so many legitimate projects flagged by the invalid repo rule probably means something is broken with the rule, but also that lowering the score was probably the most correct fix here. This particular rule isn't accurate right now and shouldn't be a primary scoring heuristic until it's fixed.

**Comment by humitos** [comment]
I agree with @agjohnson here. I wouldn't complicate this too much. Lowering down the score is probably enough for now.

We want to create more rules with low score and have less with high scores. That way, the project marked as spam is more likely to be spam.

**Comment by ericholscher** [comment]
I already lowered the score, and I'm agreed that actual spam users shouldn't access the project. It's definitely bad for false positives tho 🙃

**Comment by stsewd** [comment]
We went a little further, and now all actions are blocked for projects marked as spam. I guess we can close this?

**Comment by agjohnson** [comment]
Yeah I think so. The threshold for blocking dashboard access to projects is high enough that I think false positives are not a strong concern. If we do notice users contacting support about projects misflagged and the dashboard blocked, we might instead only block some views or POST requests instead.
